### PR TITLE
Removed menu from identity service

### PR DESCRIPTION
**Pull request by mick-jones**
- Removed menu from identity service as per conversation with Saadia and Abi. This menu had nothing in it and a user should not be able to navigate the site until logged in
- Added Sign in p tag in plave of menu
- Added more padding to page bottom on mobile devices to show forgotten password link

### CODE REVIEW

File: src/main/resources/static/assets/css/main.css [hunk 1]
```diff
@@ -5050,7 +5059,7 @@ hr.clear {
   top: 10px;
 }
 
-.autocomplete__menu {
+.autocomplete__menu {
```
Comment (elxsy) [inline src/main/resources/static/assets/css/main.css]: space?

File: src/main/resources/templates/fragments/layout.html [hunk 2]
```diff
@@ -14,6 +14,7 @@
     <body>
     <nav lang="en" th:replace="fragments/nav :: nav"> </nav>
         <div class="container">
+            TYest
```
Comment (elxsy) [inline src/main/resources/templates/fragments/layout.html]: mistake?

File: src/main/resources/templates/fragments/user-header.html [hunk 3]
```diff
@@ -11,8 +11,7 @@
             </div>
             <div class="header-proposition">
                 <div class="content">
-                    <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
-                    <nav lang="en" th:replace="fragments/user-nav :: nav"> </nav>
+                    <p class="sign-in">Sign in</p>
```
Comment (elxsy) [inline src/main/resources/templates/fragments/user-header.html]: you have completely removed the navigation?

File: src/main/resources/static/assets/css/main.css (hunk 1 above)
Comment (mick-jones) [reply src/main/resources/static/assets/css/main.css]: Removed

File: src/main/resources/templates/fragments/layout.html (hunk 2 above)
Comment (mick-jones) [reply src/main/resources/templates/fragments/layout.html]: Removed

File: src/main/resources/templates/fragments/user-header.html (hunk 3 above)
Comment (mick-jones) [reply src/main/resources/templates/fragments/user-header.html]: Yes. There is no need for any navigation on this page. Once successfully logged in the user will be redirected to their domain with full navigation options. The menu was completely empty anyway.

**Pull request merged**